beamer/tests/agent/test_request.py

```diff
@@ -113,23 +113,15 @@ def test_challenge_own_claim(config, request_manager, token, direction):
     assert not maybe_challenge(claim, context), "Tried to challenge own claim"
 
 
-@pytest.mark.parametrize("allow_unlisted_pairs", (True, False))
-def test_fill_and_claim(request_manager, token, agent, allow_unlisted_pairs, direction):
+def test_fill_and_claim(request_manager, token, agent, direction):
```

Comment:
Why was the `allow_unlisted_pairs` fixture and parameterization removed?

Reply:
The reason for it was that with the `token_list` fixture, the token is allowed either way.
So the test becomes useless. Additionally, it is not essential for what the test should do. the test is about filling and claiming, not about filtering requests.

Reply:
Then we should probably also remove the `allow_unlisted_pairs` fixture from `conftest.py` as it does not appear to be used anymore.

Reply:
it is actually used in the agent via `set_allow_unlisted_pairs` and thus in a lot of tests. I test if we can get rid of this at all.

Reply:
tests passed smoothly by removing the ENV. will remove it